astroquery/alma/core.py

```diff
@@ -207,6 +208,92 @@ def _gen_sql(payload):
     return sql + where
 
 
+def get_enhanced_table(result):
+    """
+    Returns an enhanced table with quantities instead of values and regions for footprints.
+    Note that this function is dependent on the ``astropy`` - affiliated ``regions`` package.
+    """
+    try:
+        import regions
+    except ImportError:
+        print(
+            "Could not import astropy-regions, which is a requirement for get_enhanced_table function in alma"
+            "Please refer to http://astropy-regions.readthedocs.io/en/latest/installation.html for how to install it.")
```

Comment:
```suggestion
            "Could not import astropy-regions, which is a requirement for get_enhanced_table function in alma.  "
            "Please refer to http://astropy-regions.readthedocs.io/en/latest/installation.html for how to install it.")
```

you need a period and whitespace here